modules/300-prometheus/images/grafana-dashboard-provisioner/hooks/dashboard_provisioner.sh

```diff
@@ -47,6 +47,12 @@ function __main__() {
 
     title=$(slugify <<<${title})
 
+    dashboardUid=$(jq -rc '.definition | try(fromjson | .uid)' <<<${dashboard})
+    if [[ "x${dashboardUid}" == "x" ]]; then
+      dashboardUid="${dashboardUid} $(jq -rc '.uid' <<<${dashboard})"
```

Comment:
* x-check isn't actual with quotation marks.
* it is important to wrap the ${dashboard} in quotations here.
```suggestion
    if [[ "${dashboardUid}" == "" ]]; then
      dashboardUid="${dashboardUid} $(jq -rc '.uid' <<< "${dashboard}")"
```